zzzz_modular_occulus/code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -265,10 +265,26 @@
 		if(temp.dislocated == 2)
 			wound_flavor_text["[temp.name]"] += "<span class='warning'>[T.His] [temp.joint] is dislocated!</span><br>"
 		if(((temp.status & ORGAN_BROKEN) && temp.brute_dam > temp.min_broken_damage) || (temp.status & ORGAN_MUTATED))
-			wound_flavor_text["[temp.name]"] += "<span class='warning'>[T.His] [temp.name] is dented and swollen!</span><br>"
+			wound_flavor_text["[temp.name]"] += "<span class='warning'>[T.His] [temp.name] is mangled!</span><br>"
+		if(temp.germ_level > INFECTION_LEVEL_ONE && temp.germ_level < INFECTION_LEVEL_TWO)//Occulus Edit: Infection status on examine
+			wound_flavor_text["[temp.name]"] += "<span class='warning'>[T.His] [temp.name] is discolored!</span><br>"
+		else if(temp.germ_level > INFECTION_LEVEL_TWO && temp.germ_level < INFECTION_LEVEL_THREE)
+			wound_flavor_text["[temp.name]"] += "<span class='warning'>[T.His] [temp.name] is oozing pus!</span><br>"
+		else if(temp.germ_level > INFECTION_LEVEL_THREE)
+			wound_flavor_text["[temp.name]"] += "<span class='warning'>[T.His] [temp.name] is covered in decaying tissue!</span><br>"
+		if(temp.status & ORGAN_DEAD)
+			wound_flavor_text["[temp.name]"] += "<span class='warning'>[T.His] [temp.name] is necrotic</span><br>"//Occulus Edit End
```

Comment:
I think punctuation might be needed here. Either an exclamation mark or a period.

Reply:
Done, and I also made it a danger span since necrotic tissue is something that will kill you, and kill you fast.